deid/dicom/pixels/clean.py

```diff
@@ -233,17 +244,35 @@ def animate(i):
         else:
             bot.warning("use detect() --> clean() before saving is possible.")
 
-    def save_dicom(self, output_folder=None, image_type="cleaned"):
+    def save_dicom(
```

Comment:
The save function should be agnostic to the file it is being given (clean or not) - so the `image_type` doesn't make sense here. Any logic for naming the file should be in the calling function.

Reply:
Let's try for that.

Reply:
At first it looks like changing this could be a breaking change. Someone downstream could be using `DicomClient().save_dicom(image_type="original")` (or more likely `save_png(image_type="original")`. But best I can tell, `self.original` is at least currently not an attribute. So only thing this breaks is non-working code? ... maybe :shrug: